build.fsx

```diff
@@ -39,7 +39,7 @@ Target.create "LspTest" (fun _ ->
       //         Timeout = TimeSpan.FromMinutes 15. })
       id
       "run"
-      """-c Release --no-build -p "./test/FsAutoComplete.Tests.Lsp/FsAutoComplete.Tests.Lsp.fsproj" -- --fail-on-focused-tests --debug"""
+      """-c Release --no-build -p "./test/FsAutoComplete.Tests.Lsp/FsAutoComplete.Tests.Lsp.fsproj" -- --debug"""
```

Comment:
`--fail-on-focused-tests` is a pretty good setting for CI

Reply:
yeah, added it back just now. I removed it for my own tight-cycle of tests, though I suppose I could just as easily do `dotnet run -- --debug` in the LSPTest directory